lib/internal/streams/utils.js

```diff
@@ -119,14 +119,14 @@ function isReadableFinished(stream, strict) {
 function isReadable(stream) {
   if (stream && stream[kIsReadable] != null) return stream[kIsReadable];
   const r = isReadableNodeStream(stream);
```

Comment:
`r` value is unused if `typeof stream?.readable` is not a boolean, it would be worth move its assignment below. (Same in `isWritable`)